app/graphql/types/freak_type.rb

```diff
@@ -13,6 +13,7 @@ class FreakType < Types::BaseObject
     field :projects, [ProjectType], null: false
     field :role, RoleType, null: false
     field :level, LevelType, null: false
+    field :level_id, GraphQL::Types::ID, null: true
```

Comment:
Why do we need to expose the id to the client, if we already expose the entire level object?